t/data/openqa-trigger-from-obs/script/rsync.sh

```diff
@@ -13,6 +13,6 @@ function sleep_until_file_created {
 [[ "$1" =~ MockProjectLongProcessing* ]] && { sleep_until_file_created "$1" || exit 1; }
 [ "$1" != MockProjectError ] || {
     echo >&2 "Mock Error"
-    exit 1
+    exit 777
```

Comment:
Exit code never seems to be propagated? I tried TDD as usual to confirm the behavior, but this only helped me understand the retries. I don't know how errors would be reported. [Docs](https://docs.mojolicious.org/Minion/Job#retry) provide no clues either.

Reply:
exit codes should be between 0 and 255